contrib/babelfishpg_tsql/src/collation.c

```diff
@@ -388,18 +398,28 @@ transform_from_ci_as_for_likenode(Node *node, OpExpr *op, like_ilike_info_t like
 	if (IsA(leftop, Const) || !IsA(rightop, Const) ||
 		((Const *) rightop)->constisnull)
```

Comment:
Why do we need to explicitly collate even if the right operand is NULL? How does NULL value work w.r.t collations